lib/sparse/src/index/inverted_index/inverted_index_compressed_immutable_ram.rs

```diff
@@ -101,13 +103,18 @@ impl<W: Weight> InvertedIndex for InvertedIndexCompressedImmutableRam<W> {
         Ok(InvertedIndexCompressedImmutableRam {
             postings,
             vector_count: ram_index.vector_count,
+            total_sparse_size: ram_index.total_sparse_size,
```

Comment:
Compressed index may apply quantization, which should be reflected in index size. So I propose to compute just number of elements in mutable ram index, and adjust it according to `size_of::<W>()` on the getter implementation.

Reply:
I just found out there is a handy `CompressedPostingListStoreSize`. I used it in the new commits.

Shout-out to @xzfc 🙌